### PR TITLE
Update configure-docker.md

### PR DESCRIPTION
**Pull request by nvtkaszpir**
Bumpt container image to 7.2.x

**What this PR does / why we need it**:

Docs update.

**Which issue(s) this PR fixes**:
None, just noticed it doing my own private docker-compose and just got triggered in the October!

Fixes #
None.

**Special notes for your reviewer**:

Not tested in given setup, but it realllllyyy works on my machine :D

### CODE REVIEW

File: docs/sources/administration/configure-docker.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ mkdir data # creates a folder for your data
 ID=$(id -u) # saves your user id in the ID variable
 
 # starts grafana with your user id and using the data folder
-docker run -d --user $ID --volume "$PWD/data:/var/lib/grafana" -p 3000:3000 grafana/grafana:5.1.0
+docker run -d --user $ID --volume "$PWD/data:/var/lib/grafana" -p 3000:3000 grafana/grafana:7.2.1
```
Comment (zoltanbedi) [inline docs/sources/administration/configure-docker.md:41]:
```suggestion
docker run -d --user $ID --volume "$PWD/data:/var/lib/grafana" -p 3000:3000 grafana/grafana
```
Would it make sense to remove the version from here?

Comment (nvtkaszpir) [reply docs/sources/administration/configure-docker.md:41]: that would force people to use 'latest' version, which is not really that safe, It's better to run specific version of the container.

Comment (nvtkaszpir) [reply docs/sources/administration/configure-docker.md:41]:
```  grafana:
    image: grafana/grafana:7.2.1
    container_name: grafana
    ports:
      - 3000:3000
    volumes:
      - ./grafana/config:/etc/grafana:ro
      - ./grafana/data:/var/lib/grafana:rw
```

Just tested wit above config with `grafana.ini` being an empty file, works :D

**Review by zoltanbedi — APPROVED** [review]
lgtm

**Comment by oddlittlebird** [comment]
Thanks for contributing to the docs!

**Pull request merged**